.buildkite/pipeline.yml

```diff
@@ -172,7 +172,7 @@ steps:
 
   - label: "coverage-x86"
     commands:
-      - pytest rust-vmm-ci/integration_tests/test_coverage.py
+      - find . -type f -name "test_coverage.py" | xargs pytest
```

Comment:
The changes to the pipeline should be in a separate commit and we should explain in the commit message why are they needed.

Reply:
Done.